### PR TITLE
Other tab fixes

### PR DESCRIPTION
**Pull request by RossKen**
-Add tooltips to Topic and member pages
-Add wordcloud to Member page (& integrating Sam's code from MPWeb)
-Rearrange Topic & Member pages

### CODE REVIEW

File: global.R [hunk 1]
```diff
@@ -23,9 +27,9 @@ myColClasses <- c("Date" = "R_date",
 
 rawData <- read.csv("./Data/MoJwrittenPQs.csv", colClasses = myColClasses)
 data <- data.frame(rawData)
-# Topic <- data$Cluster
-# Topic_Keywords <- data$Cluster_Keywords
-# data <- cbind(data, Topic, Topic_Keywords)
+drops <- c("X","Document_Number", "Corrected_Date")
```
Comment (EdwardAndress) [inline global.R]: Corrected date will no longer feature in the data, after my PR goes in.  X should also be gone.  If we're not using Document number perhaps we should get rid of that too...

File: global.R [hunk 2]
```diff
@@ -84,3 +88,14 @@ queryVec <- function(query){
     })
   return(which(vocab %in% query))
 }
+
+plotWordcloud <- function(analysisObject){
+  words <- analysisObject$Questions$Question_Text %>%
```
Comment (EdwardAndress) [inline global.R]: Is this the same clean up that's done elsewhere?

File: global.R (hunk 1 above)
Comment (RossKen) [inline global.R]: I want to keep it in for the 2nd and 3rd tab to give as much information as possible

File: global.R (hunk 2 above)
Comment (RossKen) [inline global.R]: Clean up?

This is the wordcloud fn Sam wrote for showing MP wordclouds

**Comment by EdwardAndress** [comment]
-- Once my branch is merged!

**Comment by RossKen** [comment]
Yea I figured there will be conflicts - should be fine though

If Travis ever finishes building!

**Comment by EdwardAndress** [comment]
We can get it done if you're gone by the time it's passing.

File: global.R (hunk 1 above)
Comment (RossKen) [inline global.R]: I think there is a way to get round not doing this, but will have to look at when I get back next week - want to get this version up and running on the platform before heading off

**Comment by RossKen** [comment]
@EdwardAndress 
Have had to make some changes as build was failing - some of the .rda files had been lost in the rebase to merge with master. Not going to have time to wait about for Travis to finish this evening, so if this could be merged and deployed for the demo on Tuesday that would be great. Will Slack you the deployment stuff Robin sent me through

**Pull request merged**